src/EnergyPlus/WindowEquivalentLayer.cc

```diff
@@ -8241,14 +8243,20 @@ namespace WindowEquivalentLayer {
 		// DERIVED TYPE DEFINITIONS
 		// na
 
-		// FUNCTION LOCAL VARIABLE DECLARATIONS:
-		Real64 RAT;
-		// Flow
+		//// FUNCTION LOCAL VARIABLE DECLARATIONS:
+		//Real64 RAT;
+		//// Flow
 
 		// TODO handle vert blind cases etc
-		RAT = L.S * std::cos( OMEGA_DEG ) / L.W;
-		// limit upward slat angle to horiz = max visibility
-		VB_CriticalSlatAngle = max( 0.0, RadiansToDeg * std::asin( RAT ) - OMEGA_DEG );
+
+		// issue #5750, commented out because it does not block the solar beam
+		//RAT = L.S * std::cos( OMEGA_DEG ) / L.W;
+		//// limit upward slat angle to horiz = max visibility
+		//VB_CriticalSlatAngle = max( 0.0, RadiansToDeg * std::asin( RAT ) - OMEGA_DEG );
```

Comment:
Should just clean these out.  With our version control we can always look back and see where the file was modified, and with good commit messages we can see why.